LEAF_Request_Portal/scripts/firstRun.php

```diff
@@ -12,8 +12,8 @@
 if (count($res) == 0) {
     if (strlen(DATABASE_DB_ADMIN) > 0) {
         $vars = array(':name' => DATABASE_DB_ADMIN);
-        $res = $db->prepared_query('INSERT INTO users (userID, groupID)
-                                        VALUES (:name, 1)', $vars);
+        $res = $db->prepared_query('INSERT INTO users (userID, groupID, backupID)
```

Comment:
This isn't best practices.  We should probably modify the database so it has a default of empty-string from that column.  Otherwise, somewhere down the road, nulls will start to creep in.

Reply:
Default of '' added